tests/integ/modin/groupby/test_groupby___iter__.py

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+import modin.pandas as pd
+import pandas as native_pd
+import pytest
+
+import snowflake.snowpark.modin.plugin  # noqa: F401
+from tests.integ.modin.utils import assert_frame_equal, assert_series_equal
+from tests.integ.utils.sql_counter import sql_count_checker
+
+
+@sql_count_checker(query_count=3, join_count=4)
+@pytest.mark.parametrize("by", ["a", ["a"]])
```

Comment:
Can you add a test with multiple `by` columns as well? You might also need to make the data a little larger.

Reply:
Done.